Dockerfile

```diff
@@ -18,58 +18,63 @@ LABEL org.opencontainers.image.source https://github.com/helmfile/helmfile
 RUN apk add --no-cache ca-certificates git bash curl jq openssh-client
 
 ARG HELM_VERSION="v3.10.2"
+ENV HELM_VERSION="${HELM_VERSION}"
```

Comment:
why add this env? I have saw `ARG HELM_VERSION="v3.10.2"`.

Reply:
@itscaro  remark , it was hidden in the resolved conversation https://github.com/helmfile/helmfile/pull/519/files#r1021514974 :)